src/types/onyx/Policy.ts

```diff
@@ -610,6 +968,9 @@ type Connections = {
 
     /** Xero integration connection */
     xero: Connection<XeroConnectionData, XeroConnectionConfig>;
+
+    /** NetSuite integration connection */
+    netsuite: NetSuiteConnection;
```

Comment:
I had the same issue that we can't reuse `Connection` object because the Netsuite doesn't match with the other existing integrations.